src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function parseSalary(salary) {
+  return Number(salary.replace('$', '').replace(',', ''));
+}
+
+function sortList(ulElement) {
+  const items = Array.from(ulElement.children);
+
+  items.sort((a, b) => {
+    const salaryA = parseSalary(a.dataset.salary);
+    const salaryB = parseSalary(b.dataset.salary);
+
+    return salaryB - salaryA;
+  });
+
+  items.forEach((item) => ulElement.appendChild(item));
```

Comment:
Appending each item back to `ulElement` in the `forEach` loop may not reorder the elements as expected. Consider using `ulElement.innerHTML = '';` before the loop to clear the list, then append each sorted item to ensure the order is updated correctly.